pkg/3scale/amp/operator/apicast_reconciler.go

```diff
@@ -739,3 +746,14 @@ func Apicast(apimanager *appsv1alpha1.APIManager, cl client.Client) (*component.
 	}
 	return component.NewApicast(opts), nil
 }
+
+func GetSystemRedisSecret(apimanagerNs string, ctx context.Context, client client.Client) bool {
+	backendRedisSecret := &v1.Secret{}
+	client.Get(ctx, types.NamespacedName{
```

Comment:
```suggestion
	err := client.Get(ctx, types.NamespacedName{
```
We should add error handling here to be safe.